README.md

```diff
@@ -1,7 +1,133 @@
 # lungair-fhir-server
 
-*Under development.*
+This repository contains tools for building a prepopulated FHIR R4 server,
+intended for the construction of sandbox environments for Electronic Health
+Records (EHR). `lungair-fhir-server` creates a [FHIR](https://www.hl7.org/fhir/overview.html) server inside a
+docker container that you can interact with. The FHIR server contains Patients
+and Observations from a user defined data source. Currently,
+a handful of observation types are supported such as heart rate and some mechanical ventilation parameters.
+The full list of supported observation types can be found in [observation_types.json](observation_types.json).
 
-This repository will contain tools for building a prepopulated FHIR R4 server,
-intended for the construction of sandbox environments.
-It will build off of the [hapi r4 container](https://hub.docker.com/layers/hapi-5/smartonfhir/hapi-5/r4-empty/images/sha256-42d138f85967cbcde9ed4f74d8cd57adf9f0b057e9c45ba6a8e1713d3f9e1cea?context=explore) by the SMART on FHIR team.
+## Initial Setup
+
+1. Ensure that [Docker](https://docs.docker.com/get-docker/) and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) are installed.
+2. Clone this project.
+    ```sh
+    git clone git@github.com:KitwareMedical/lungair-fhir-server.git
+    ```
```

Comment:
An additional setup step we need here is _setting up the python environment_.

@stephencrowell I recommend:
- creating a venv (with python version 3.9 or earlier)
- pip installing to it _only the things that seem necessary_ for this project
- activating the venv and testing this project to make sure it runs
- `pip freeze > requirements.txt`
- commit the requirements file to this branch
- add `pip install -r requirements.txt` to the instructions here